### PR TITLE
[doc][prover] add doc for installing Move prover dependencies

### PR DESCRIPTION
**Pull request by rahxephon89**
### Description

This PR updates the doc on how to install dependencies for Move prover.

### Checklist

- Do all Lints pass?
  - [x] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx [hunk 1]
```diff
@@ -3,6 +3,20 @@ import { Callout } from 'nextra/components'
 # Install the Move Prover
 
 If you want to use the [Move Prover](../../smart-contracts/prover.mdx), install the Move Prover dependencies after [installing the CLI binary](.).
+There are two ways to install Prover dependencies.
+
+## Installation through Aptos CLI (Recommended)
+
+1. Install the latest Aptos CLI binary.
```
Comment (vineethk) [inline apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx:10]: Might be nice to link this text to a page with instructions to install the latest Aptos CLI binary.

File: apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx [hunk 2]
```diff
@@ -3,6 +3,20 @@ import { Callout } from 'nextra/components'
...
+
+2. Execute the command `aptos update prover-dependencies`.
+
+<Callout type="info">
+  Environment variable `BOOGIE_EXE` and `Z3_EXE` will be set automatically after installation. Please make sure
```
Comment (vineethk) [inline apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx:15]:
```suggestion
  Environment variables `BOOGIE_EXE` and `Z3_EXE` will be set automatically after installation. Please make sure
```

File: apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx [hunk 3]
```diff
@@ -27,7 +41,8 @@ If you want to use the [Move Prover](../../smart-contracts/prover.mdx), install
 
    </details>
 
-3. You can now run the Move Prover to prove an [example](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/hello_prover):
+
+After installation, you can now run the Move Prover to prove an [example](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/hello_prover):
```
Comment (vineethk) [inline apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx:45]:
```suggestion
After installation, you can run the Move Prover to prove an [example](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/hello_prover):
```

**Review by vineethk — APPROVED** [review]

File: apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx (hunk 1 above)
Comment (rahxephon89) [reply apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx]: done

File: apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx (hunk 2 above)
Comment (rahxephon89) [reply apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx]: done

File: apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx (hunk 3 above)
Comment (rahxephon89) [reply apps/nextra/pages/en/build/cli/setup-cli/install-move-prover.mdx]: done

**Review by wrwg — APPROVED** [review]

**Pull request merged**